src/app/lib/constants/styles.js

```diff
@@ -24,13 +24,14 @@ export const FF_NEWS_SERIF_REG = `ReithSerifNewsRegular${fontFamilyBase}`;
 export const FF_NEWS_SERIF_MDM = `ReithSerifNewsMedium${fontFamilyBase}`;
 export const FF_NEWS_SERIF_BLD = `ReithSerifNewsBold${fontFamilyBase}`;
 
-/* 
+/*
     GEL Spacing
-    Assumes 16px is set as the default font-size. 
+    Assumes 16px is set as the default font-size.
     This is changeable in the user's browser settings, as the html font-size is 100%
 */
 export const GEL_SPACING = `0.5rem`;
 export const GEL_SPACING_DBL = `1rem`; // 16px
+export const GEL_SPACING_TRPL = `1.5rem`; // 16px
```

Comment:
Can you please remove the comment `// 16px` from this line?

Reply:
The line above shouldn't be `16px` either as it might not be `16`.

Reply:
This is a copy paste mistake, will remove